services/outputhost/ackmanager.go

```diff
@@ -65,12 +59,11 @@ type (
 		cgUUID             string
 		extUUID            string
 		connectedStoreUUID *string
-		*levels                    // ‡ the current levels
-		prev               *levels // ‡ the previous levels
+		*levels            // ‡ the current levels
 		ackLevelTicker     *time.Ticker
 		closeChannel       chan struct{}
 		waitConsumed       chan<- bool // waitConsumed is the channel which will signal if the extent is completely consumed given by extentCache
-		metaclient         metadata.TChanMetadataService
+		committer          Committer
```

Comment:
The point of doing the committer was to try not to change/duplicate the Ackmanager, which has historically had difficult to find bugs. You aren't wrong, but what we'll have is basically 2 ack managers at that point.